examples/express-javascript-live-cursors/static/app.js

```diff
@@ -0,0 +1,1977 @@
+(() => {
```

Comment:
This file should not be committed. Is it because the .gitignore is missing?

Reply:
Fixed